src/Engine/ProtoCore/DSASM/DSArray.cs

```diff
@@ -53,52 +53,6 @@ public void CollectElementsForGC(Queue<StackValue> gcQueue)
             }
         }
 
-        /// <summary>
-        /// Returns true if array contain key or not.
-        /// </summary>
-        public bool ContainsKey(StackValue key)
```

Comment:
@aparajit-pratap do you think this could cause binary compatibility issues for anyone who has references to this dll - what about just obsoleting these? They are public methods...

Reply:
Will obsolete.